client/src/components/Feed/Post.js

```diff
@@ -521,7 +521,6 @@ const Post = ({
           />
         </>
       ) : (
```

Comment:
```suggestion
      ) : (
        //Post in feed.
```

I don't think we should remove this comment - it's still helpful.

Reply:
I will just reset this file to staging & repush

Reply:
I haven't realised & also do not remember that I removed this comment. anyway, I agree.